submissions/OlStaniev/DOM/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700;900&display=swap" rel="stylesheet">
+    <title>Star Wars Saga</title>
+</head>
+
+<body>
+    <header class="header">
+        <img src="./img/star-wars.svg" alt="star wars logo" class="header__logo">
+        <button class="header__burgerMenu"></button>
+    </header>
+    <main>
+        <nav class="nav">
+            <ul class="nav__list">
+                <!-- Test text -->
+                <!-- <li class="listItem">
+                    <button class="nav__list_item" data-id="ep4">
+                        <span class="listTitle">Episode IV</span>
```

Comment:
Leave comments in the test or debug thread. In the final code version, they are not needed or used very rarely.